test/compile/mark/area.test.ts

```diff
@@ -49,4 +49,38 @@ describe('Mark: Area', function() {
       assert.deepEqual(props.fill, {scale: COLOR, field: 'Miles_per_Gallon'});
     });
   });
+  
```

Comment:
All the old examples in the vega-editor do work. But I didn't know there were no tests for the old code. I can quickly write those tests.